libs/constrained-generators/test/Constrained/Test.hs

```diff
@@ -5,11 +5,13 @@
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE QuasiQuotes #-}
```

Comment:
Nothign has changed in this module. Are these added extensions really necessary?
```suggestion
```